SUMMARY.md

```diff
@@ -442,6 +448,7 @@
 
 - [Contributing to Hardware](./doc/contributing/hw/README.md)
   - [Comportability](./doc/contributing/hw/comportability/README.md)
+    - [Comportable IP (CIP)  Testbench Architecture](./hw/dv/sv/cip_lib/README.md)
```

Comment:
This page is a duplicate of the page in the 'Contributing to Verification'. I think it would make sense to remove this one. It may also make sense for it to stay in the 'SystemVerilog and UVM Components' section and be linked to from the contributing sections.

Reply:
Yeah so I deliberately put links to this page in two places because it feels to me at least that it has multiple homes. I think it fits well 'next-to' the Comportability specification, but also equally in the 'Contributing to DV' section and probably also the "SV/UVM" section too.
Do you think having multiple links to the same page is too confusing? I'm undecided.

Reply:
Hmmm I'm unsure as well. @jwnrt and @marnovandermaas any thoughts?